bigtable/client/client_options.h

```diff
@@ -31,11 +33,22 @@ inline namespace BIGTABLE_CLIENT_NS {
 class ClientOptions {
  public:
   ClientOptions();
-  const std::string& endpoint() const { return endpoint_; }
-  ClientOptions& SetEndpoint(const std::string& endpoint) {
-    endpoint_ = endpoint;
+
+  /// Return the current endpoint for data RPCs.
+  const std::string& data_endpoint() const { return data_endpoint_; }
+  ClientOptions& SetDataEndpoint(std::string endpoint) {
```

Comment:
Should this not be `set_data_endpoint()` since it's a trivial setter method? Same for `set_admin_endpoint` below.

CamelCase implies a non-trivial / expensive implementation.

Reply:
Done.

And that is one of the wrong things with the Google Style Guide naming conventions, they leak details about the implementation in the name of the functions.